### PR TITLE
Exclude PFProduct, PFPush internal things from watchOS target.

### PR DESCRIPTION
**Pull request by nlutsenko**
Contributes to #179.

### CODE REVIEW

File: Parse/Internal/Commands/PFRESTPushCommand.h [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface PFRESTPushCommand : PFRESTCommand
+PF_WATCH_UNAVAILABLE @interface PFRESTPushCommand : PFRESTCommand
```
Comment (richardjrossiii) [inline Parse/Internal/Commands/PFRESTPushCommand.h]: Please do pus related items in a separate PR.

Comment (nlutsenko) [inline Parse/Internal/Commands/PFRESTPushCommand.h]: Renamed and rebased.
